controllers/grid/users/reviewer/ReviewerGridHandler.php

```diff
@@ -39,7 +40,6 @@ public function reviewRead($args, $request)
             $submission = $this->getSubmission();
             $reviewer = Repo::user()->get($reviewAssignment->getReviewerId(), true);
             $user = $request->getUser();
-            import('classes.log.SubmissionEventLogEntry'); // Constants
```

Comment:
Unfortunately the `SUBMISSION_LOG_...` constants defined in `classes/log/SubmissionEventLogEntry.php` have not been moved inside the class yet, so until then, this line is best to keep.

Reply:
Re-added .